NavigationMenu.md

```diff
@@ -46,9 +46,13 @@
   * [Information extraction](./guides/V1_wwsg_InformationExtraction.md)
   * [Information extraction annotations](./guides/V1_Annotation_Message_Information_Extraction.md)
 
+* ## [Actions and Events](./guides/actions_and_events_overview.md)
```

Comment:
this section should be only
*Actions and Events
*Client action handler callback API*
*Get action context information*

Reply:
Modified based on your comments. Thanks!

Reply:
I link the "Actions and Events" section to your topic: V1_ActionsEvents.md.